types/between-tests.ts

```diff
@@ -22,7 +22,7 @@ between.on('complete', (value) => { value });
 // Test pause control
 between.pause();
 between.start();
```

Comment:
@hirako2000 How about renaming `.start()` to `.run()` ?

Reply:
I called it `start` because the emitted event that was already present is called `start`. It's for consistency.
But up to you, I can rename if you think it's better.